src/styles/components/pages/Offer/StocksManager/_StocksManager.scss

```
@@ -33,6 +27,17 @@
     }
   }
 
+  .notification {
+    max-height: 40px;
```

```suggestion
    max-height: rem(40px);
```
Pour améliorer l'accessibilité